app/views/about/privacy.en.html.erb

```diff
@@ -1,155 +1,154 @@
-<% title 'Privacy policy' %>
+<% title 'Privacy notice' %>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <h1 class="govuk-heading-xl">Privacy policy</h2>
-    <h2 class="govuk-heading-l">Purpose</h2>
-    <p class="govuk-body">This privacy notice sets out:</p>
-    <div class="govuk-list">
-      <ul class="govuk-list govuk-list--bullet">
-        <li>the standards that you can expect from the Legal Aid Agency (LAA) when we request or hold personal data about you</li>
-        <li>how you can get access to a copy of your personal data</li>
-        <li>what you can do if you think the standards are not being met</li>
+    <h1 class="govuk-heading-xl">Privacy notice</h2>
+    <h2 class="govuk-heading-l">Apply for Criminal Legal Aid Service</h2>
+    <p class="govuk-body">This service is provided by the
+      <a class="govuk-link" rel="external" target="_blank" href="https://www.gov.uk/government/organisations/legal-aid-agency/">Legal Aid Agency (LAA)</a>,
+      part of the Ministry of Justice (MoJ).
+    </p>
+
+    <p class="govuk-body">This privacy notice covers how the digital service (“Apply for criminal legal aid”) collects personal data from providers who are submitting information on behalf of their clients.</p>
+
+    <p class="govuk-body">
+      For details of how the Legal Aid Agency collects and manages the personal data submitted within the application, check the
+      <a class="govuk-link" rel="external" target="_blank" href="<%= Settings.portal_help_info_url %>">Legal Aid Agency privacy notice</a>.
+    </p>
+
+    <h2 class=" govuk-heading-l ">What data we collect</h2>
+
+    <p class=" govuk-body ">We collect:</p>
+    <div class=" govuk-list ">
+      <ul class=" govuk-list govuk-list--bullet ">
+        <li>
+          your portal login and office account number provided by the LAA Portal service
+          (<a class=" govuk-link " rel=" external " target=" _blank " href="<%= Settings.portal_help_info_url %>">LAA Portal terms and conditions</a>)
+        </li>
+        <li>
+          your Internet Protocol (IP) address
+        </li>
+        <li>
+          details of which version of
+          <a class=" govuk-link " rel=" external " target=" _blank " href=" https: www.gov.uk support browsers ">web browser</a>
```

Comment:
There are extraneous spaces and the href is invalid here.

Reply:
One of the linters has done this and I didn't notice, sorry!